### PR TITLE
documented `false` as a allowed `scrollTo` value

### PR DESCRIPTION
**Pull request by staabm**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -165,7 +165,7 @@ key | default | description
 `replace` | false | replace URL without adding browser history entry
 `maxCacheLength` | 20 | maximum cache size for previous container contents
 `version` | | a string or function returning the current pjax version
-`scrollTo` | 0 | vertical position to scroll to after navigation
+`scrollTo` | 0 | vertical position to scroll to after navigation. `false` to disable scrolling.
```
Comment (mislav) [inline README.md]: Thanks. Can it say "false to not change scroll position"? Because we aren't really "disabling" scrolling.

Comment (mislav) [inline README.md]: Something like: "To avoid changing scroll position, pass `false`."

**Comment by staabm** [comment]
@mislav fixed per feedback

**Comment by staabm** [comment]
@mislav fixed per feedback

**Comment by staabm** [comment]
hmm seems like a github bug (double post and close)

**Comment by mislav** [comment]
Looks great thanks. Can you remember what you did when you got double-post and closed by accident? Did you press the "Comment" button with mouse or used <kbd>Cmd+Enter</kbd>? I can try fixing the bug if you gave me more information about how to reproduce.

**Comment by staabm** [comment]
I did a ctrl+enter in firefox developer edition on win7x64

**Comment by mislav** [comment]
Thanks. Is there a chance you pressed <kbd>Ctrl+Enter</kbd> twice?

**Comment by staabm** [comment]
Hm that might also be the case...

**Comment by mislav** [comment]
I cannot reproduce this using Ctrl+Enter on Firefox on Windows. We disable
all form submit buttons upon first submit and until the form has finished
posting so that double submits are prevented. I can't imagine how it
happened that it closed the PR for you.

**Comment by staabm** [comment]
I am also unable to reproduce

**Pull request merged**